src/media/css/rtl/footer.styl

```diff
@@ -0,0 +1,10 @@
+@import '../lib';
+
+@media $base-tablet {
+    +rtl() {
+        .footer p {
+            padding: 0 1em 0 0;
+            margin: 0 0 0 100px;
```

Comment:
Is this for the `.footer-content` area? It might make sense to lean on flex some more there. Remove all the padding and margin from the Mozilla logo and privacy policy, etc links and put `margin: 0 30px; flex-grow: 1;` on the `p` (although it should really have a class name).